dot/digest/interface.go

```diff
@@ -18,6 +18,7 @@ type BlockState interface {
 	FreeFinalisedNotifierChannel(ch chan *types.FinalisationInfo)
 }
 
+//go:generate mockgen -destination=mock_epoch_state_test.go -package $GOPACKAGE . EpochState
```

Comment:
add an empty line so it's not part of `EpochState`'s comment

```suggestion
//go:generate mockgen -destination=mock_epoch_state_test.go -package $GOPACKAGE . EpochState

```